packages/expo-video/ios/NowPlayingArtworkCache.swift

```diff
@@ -0,0 +1,90 @@
+import UIKit
+import MediaPlayer
+import Foundation
+
+// If we ever want to use this class for something else we should probably make these configurable through the constructor
+// I couldn't find any information on recommended artwork resolution, but this seems like a reasonable value considering how it's displayed ~@behenate
+private let maxImageSize = CGSize(width: 1024, height: 1024)
```

Comment:
`1024 x 1024` seems like a lot 🤔 
If we're going to cache it, I believe we should reduce it. What do you think?